PICMI_Python/requirements.txt

```diff
@@ -1,2 +1,4 @@
 numpy~=1.15
 scipy~=1.5
+autoclass
```

Comment:
Checked: pure python (incl. dependencies) & easy to install it seems
- https://pypi.org/project/autoclass/#files
- https://github.com/smarie/python-autoclass/blob/master/pyproject.toml
- https://github.com/smarie/python-autoclass/blob/master/setup.py